Dangerfile.main.kts

```diff
@@ -21,3 +24,5 @@ if (danger.git.createdFiles.size + danger.git.modifiedFiles.size - danger.git.de
 if (danger.github!!.pullRequest.title.contains("WIP" ,false)) {
     warn("PR is classed as Work in Progress")
 }
+
+warn(WordUtils.capitalize("thisIsAPr"))
```

Comment:
You should probably remove this :P Otherwise every PR will get this message, you can use `WordUtils.capitalize("thisIsAPr")` on 
```kotlin
warn(WordUtils.capitalize("big PR, try to keep changes smaller if you can"))
```

Reply:
sure